scss/_base_typography-max-widths.scss

```diff
@@ -1,3 +1,15 @@
+@mixin vf-b-typography-max-widths {
+  %max-width--p {
+    max-width: map-get($max-widths, default);
+  }
+
+  //expose measure as a class
+  %measure--p,
```

Comment:
Why do we need two placeholders that do the same thing? `%max-width--p` and `%measure--p`

Reply:
I discovered this is used on ubuntu.com, so purely to avoid breaking. This is just reverting a deletion. To make it clearer, I've wrapped it in a deprecate mixin